store/store.go

```diff
@@ -66,6 +67,8 @@ type Store interface {
 	GetDBSchemaVersion() (int, error)
 	GetAppliedMigrations() ([]model.AppliedMigration, error)
 	GetDbVersion(numerical bool) (string, error)
+	GetInternalMasterDB() *sql.DB
+	GetInternalReplicaDBs() []*sql.DB
```

Comment:
Maybe worth adding a comment here to explain that these are added for other products in the multi-product architecture to get raw access to the DB. Just for someone wondering why these are exposed.

Reply:
Yeah, that makes sense.

Reply:
I added comments to both methods in the interface.